microservices.yml

```diff
@@ -81,6 +81,36 @@ commands:
           command: |
             make push-image-to-ecr
 
+  push-built-image-to-prod-ecr:
+    description: Publish docker image to production ECR
+    steps:
+      - install-okta-aws-cli
+      - run:
+          name: Switch to PRODUCTION aws account
+          command: |
+            rm -f ~/.okta/cookies.properties
+            rm -f ~/.okta/.current-session
+            rm -f ~/.okta/profiles
+
+            sed "s~OKTA_AWS_ROLE_TO_ASSUME=.*~OKTA_AWS_ROLE_TO_ASSUME=$OKTA_AWS_PROD_ROLE_TO_ASSUME~g" -i ~/.okta/config.properties
+            java -jar ~/.okta/okta-aws-cli.jar sts get-caller-identity
+      - run:
+          name: Push Image to production ECR
+          command: |
+            echo "export ECR_APP_IMAGE_ORIGINAL=$ECR_APP_IMAGE" >> $BASH_ENV
+            echo "export ECR_APP_IMAGE=$(aws sts get-caller-identity | jq -r '.Account').dkr.ecr.us-west-2.amazonaws.com/devices" >> $BASH_ENV
+            echo "Manually set ECR_APP_IMAGE to $ECR_APP_IMAGE_ORIGINAL for prod aws account access"
+            . $BASH_ENV
```

Comment:
you can skip exporting this instead just prepend `make` command with the value `ECR_APP_IMAGE="$(aws sts get-caller-identity | jq -r '.Account').dkr.ecr.us-west-2.amazonaws.com/devices" make build tag push-image-to-ecr` it "should" work...

then you don't need to revert those changes